mongodbatlas/fw_resource_mongodbatlas_search_deployment.go

```diff
@@ -114,6 +115,22 @@ func (r *SearchDeploymentRS) Schema(ctx context.Context, req resource.SchemaRequ
 
 const defaultSearchNodeTimeout time.Duration = 3 * time.Hour
 
+func createUpdateTimeConfig(configuredTimeout time.Duration) retrystrategy.TimeConfig {
```

Comment:
nit - just to align with the syntax below - said that, why did we use different config for the delete? why not just aligning it to all?
```suggestion
func createUpdateRetryTimeConfig(configuredTimeout time.Duration) retrystrategy.TimeConfig {
```